react/week2/todo-list-app/src/App.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import Header from './Components/Header';
+import './App.css';
+import todoArray from './Components/Mylist.json';
+import Timer from './Components/Timer';
+import Todolist from './Components/TodoLsit';
+function App() {
+  const randomList = ["do dishes", "party on next door", "Do laundry", "make food"]
+  const [toDoList, setToDoList] = useState(todoArray);
+  const addTodo = () => {
+    const randomIndex = Math.floor(Math.random() * randomList.length);
+    setToDoList([...toDoList, {
+      id: toDoList.length + 1,
```

Comment:
This is not the best approach since it can happen that you have a duplication of ids. For example, if there are three items in the list, you delete the second one and then add a new item, the new item will have id 3, the same as the third item from the list. 
You can try to find the maximum value of ids in the list and then add 1: 
Math.max(toDoList.map(item => item.id)) + 1